cygrid/cygrid.pyx

```diff
@@ -1042,7 +1059,7 @@ cdef class SlGrid(Cygrid):
         data : `~numpy.array` [2D] of float32 or float64
             The gridded spectral data for each of the input positions.
         '''
-        return self.datacube / self.weightscube
+        return self.datacube[:, 0] / self.weightscube[:, 0]
```

Comment:
Same as comment above; I don't immediately understand the slicing here. Does datacube[:, 0] extract a list of spectra?